platform/mellanox/mlnx-fw-upgrade.j2

```diff
@@ -219,7 +241,11 @@ function UpgradeASICFW() {
 function UpgradeCPLDFW_Worker() {
     local -r _CPLD_BURN_FILE="${1}"
     local -r _CPLD_REFRESH_FILE="${2}"
-    local -r _ASIC_DEV="$(find /dev/mst -iname '*_pciconf0')"
+    local -r _ASIC_DEV="$(find /dev/mst -iname '*_pci_cr0')"
```

Comment:
Some fix sneaks in togather with this change?

Reply:
@stepanblyschak yes. Now we suppose to use `_pci_cr0`, so aligned accordingly.

Reply:
Suggest to add to description/title to avoid comments additional question from reviewers